source/documentation/values/calculations.html.md.erb

```diff
@@ -258,3 +303,81 @@ combined at build time, so `min(12px % 10, 10%)` will throw an error.
     padding-right: 20px;
   }
 <% end %>
+
+### [`round()`]
+<% impl_status dart: "1.64.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  `round()` a as Sass function. To create a plain CSS calculation function
+  call for those implementations, you can write something like
+  `round(#{$number})` instead.
+
+  [special functions]: ../syntax/special-functions
+<% end %>
+
+The `round(<strategy>, value, step)` function accepts an optionl rounding strategy, 
+a value to be rounded and a rounding interval `step`. 
+`strategy` should be `nearest`, `up`, `down`, or `to-zero`.
+`round(value)` is also supported for backwards compatibility.
```

Comment:
It's probably a good idea to encourage users to supply a `step` parameter rather than relying on this backwards-compatibility.

Reply:
Hm maybe I should just delete this sentence to not encourage users to use a single argument round? Since the rest of the paragraph mentions a required value and step.